### ISSUE TITLE
Access to keyboardDisplayRequiresUserAction

### ISSUE THREAD

**Issue by hadynz** [alinz/react-native-webview-bridge#120]
I am trying to set initial focus on an input text field that is loaded in a HTML page that is rendered inside my react native webview. 

It seems that a prerequisite for this to be done is to set [keyboardDisplayRequiresUserAction][1] to `false` on the webview.

Given this is not supported as of yet by either react native's webview or webview-bridge, wondering if there is a workaround for direct access to the native module that other people might be using.

I would make a pull request with the change myself, if I wasn't afraid of the steep learning curve to delve into Objective-C land.

Thoughts?

[1]: https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIWebView_Class/#//apple_ref/occ/instp/UIWebView/keyboardDisplayRequiresUserAction

**Comment by walter0331** [comment]
+1

**Comment by alinz** [comment]
@hadynz and @walter0331 please checkout from master. @amsul fixed the issue and his fix is already merged to master. Please close this ticket if it's fixed.

**Comment by hadynz** [comment]
@alinz have just tested the master branch and can confirm that it works as expected 👍 

This change was not part of the latest release, so I did have to test this from master. Worth bumping versions so it's available for all.

For the benefit of others reading this thread, if you are loading a source page with an input field that **has focus on page load**, and set the `WebViewBridge` property `keyboardDisplayRequiresUserAction` to `false` then the keyboard will be displayed as documented by apple [here][1].

```js
<WebViewBridge
  source={{url}}
  keyboardDisplayRequiresUserAction={false}
  />
```

Closing this to mark as resolved.

[1]: https://developer.apple.com/reference/uikit/uiwebview/1617967-keyboarddisplayrequiresuseractio
